pygments/lexers/smithy.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Smithy IDL lexer
+~~~~~~~~~~~~~~~~
+
+Lexers for the Smithy IDL.
+"""
```

Comment:
Missing the license blurb. All Pygments contributions are under BSD and the file header should mention that.

Reply:
Added.